src/main/java/io/cdap/plugin/gcp/publisher/source/PubSubInputDStream.java

```diff
@@ -0,0 +1,58 @@
+package io.cdap.plugin.gcp.publisher.source;
```

Comment:
Missing copyright and license header.

Reply:
Oops, I had to exchange my Mac for a new computer and lost the IDE setting to generate the header. I'll fix that.